### ISSUE TITLE
[Feature] ChitChat integration or similar feature

### ISSUE THREAD

**Issue by gbdlin** [night/betterttv#4671]
tl;dr If you don't know what [ChitChat](https://chitchat.ma.pe/) is, divides chat into 2 separate windows: chit (spam, unnecessary clutter) and chat (actual messages worth reading). 

I would love to see the integration with ChitChat at some point in time. Currently it is possible to replace the chat panel in Twitch with an ChitChat iframe, but that'll mean losing some features of BetterTTV. Is it possible to somehow integrate with that addon, so the features persist? Maybe a similar feature can be implemented in BetterTTV itself (it then should be possible to enable it per-channel basis as this isn't really useful in smaller twitch channels)?
